docs/eventing/event-delivery.md

```diff
@@ -72,14 +72,14 @@ after the specified number of retries.
 
 ## Common Delivery Parameters
 
-The `delivery` value must be a [Delivery Spec](https://pkg.go.dev/knative.dev/eventing/pkg/apis/duck/v1?tab=doc#DeliverySpec)
+The `delivery` value must be a Delivery Spec.
 
 ### deadLetterSink
 
 When present, events that failed to be consumed are sent to the `deadLetterSink`.
 In case of failure, the event is dropped and an error is logged into the system.
 
-The `deadLetterSink` value must be a [Destination](https://pkg.go.dev/knative.dev/pkg/apis/duck/v1#Destination).
+The `deadLetterSink` value must be a Destination.
```

Comment:
Can you update the example below (line 87) by replacing `<Destination>` with the below YAML?
```YAML
# DeadLetterSink is the sink receiving event that could not be sent to
# a destination.
deadLetterSink:
  ref:
    apiVersion: v1
    kind: Service
    name: my-service
  uri: /my-path
```

Reply:
Removed the existing yaml with the new one.